ctru-rs/src/services/cfgu.rs

```diff
@@ -0,0 +1,227 @@
+//! Configuration service
+//!
+//! This module contains basic methods to retrieve and change configuration from the console.
+
+#[derive(Debug)]
+#[repr(u8)]
+pub enum Region {
```

Comment:
You should reference the `ctru_sys` enums rather than hard-coding numbers. (Here and in the conversion methods).

Reply:
(I think) this is resolved now